File: 13300.py
a, K = map(int, input().split())

arr = [[0] * 6 for _ in range(2)]

for _ in range(a):
    S, Y = map(int, input().split())
    arr[S][Y - 1] += 1

room = 0
for i in range(2):
    for j in range(6):
        if (arr[i][j] > 0):
            room += (arr[i][j] + K - 1) // K
print(room)

File: CPP/13300.cpp
#include <iostream>
using namespace std;

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(0);
    int N, K, S, Y;
    int arr[2][6] = {};
    cin >> N >> K;
    for (int i = 0; i < N; i++){
        cin >> S >> Y;
        arr[S][Y - 1]++;  
    }
    int room = 0;
    for (int i = 0; i < 2; i++){
        for (int j = 0; j < 6; j++){
            if (arr[i][j] == 0) continue;
            room = room + (arr[i][j] + K - 1) / K;
        }
    }
    cout << room;
    return 0;
}